packages/lpa-questionnaire-web-app/tests/unit/validators/accuracy-submission.test.js

```diff
@@ -0,0 +1,113 @@
+const { validationResult } = require('express-validator');
+const { testExpressValidatorMiddleware } = require('./validation-middleware-helper');
+const { rules } = require('../../../src/validators/accuracy-submission');
+
+describe('validators/accuracy-submission', () => {
+  describe('rules', () => {
+    it('has a rule for the accurate submission', () => {
+      const rule = rules()[0].builder.build();
+
+      expect(rule.fields).toEqual(['accurate-submission']);
+      expect(rule.locations).toEqual(['body']);
+      expect(rule.optional).toBeFalsy();
+      expect(rule.stack).toHaveLength(3);
+
+      expect(rule.stack[0].validator.name).toEqual('isEmpty');
+      expect(rule.stack[0].negated).toBeTruthy();
+      expect(rule.stack[0].message).toEqual(
+        'Select yes if the information accurately reflects the planning application'
+      );
+
+      expect(rule.stack[2].validator.name).toEqual('isIn');
+      expect(rule.stack[2].options[0]).toEqual(['yes', 'no']);
+    });
+
+    it('has a rule for inaccuracy reason', () => {
+      const rule = rules()[1].builder.build();
+
+      expect(rule.fields).toEqual(['inaccuracy-reason']);
+      expect(rule.locations).toEqual(['body']);
+      expect(rule.optional).toBeFalsy();
+      expect(rule.stack).toHaveLength(2);
+
+      expect(rule.stack[0].chain).toHaveProperty('if');
+      expect(rule.stack[0].chain.builder.fields).toEqual(['accurate-submission']);
+      expect(rule.stack[0].chain.builder.locations).toEqual(['body']);
+
+      expect(rule.stack[1].validator.name).toEqual('isEmpty');
+      expect(rule.stack[1].negated).toBeTruthy();
+      expect(rule.stack[1].message).toEqual(
+        'Enter details of why this does not accurately reflect the planning application'
+      );
+    });
+  });
+
+  describe('validator', () => {
+    [
```

Comment:
This test could be clearer if large examples are in separate files

Reply:
Given the shared modular nature of this test, I think it makes sense. As we are doing single questions this structure is ok, if pushing it a little. Plus it keeps us consistent with the other team.